src/vcpkg/postbuildlint.cpp

```diff
@@ -320,7 +320,30 @@ namespace vcpkg::PostBuildLint
 
         return LintStatus::SUCCESS;
     }
+#ifndef _WIN32
+    static LintStatus check_share_folder_name(const Filesystem& fs, const Path& package_dir, const PackageSpec& spec)
+    {
+        const auto cmake_folder_name = package_dir / "share";
+        std::error_code ec;
+        const auto share_folders = fs.get_directories_non_recursive(cmake_folder_name, ec);
+        for (auto&& share_folder : share_folders)
+        {
+            auto folder_name = share_folder.filename().to_string();
+            Strings::ascii_to_lowercase(folder_name.data(), folder_name.data() + folder_name.size());
 
+            if (folder_name == spec.name() && share_folder.filename() != spec.name())
+            {
+                vcpkg::printf(Color::warning,
+                              "The folder %s should be renamed %s\n",
```

Comment:
This must go through our localization machinery, should help the user understand the issue, and ideally explain how to fix it:

```
"Folders in /share must use lowercase ascii characters. Please rename it.\n"
"    file(RENAME \"${{CURRENT_PACKAGES_DIR}}/share/{orig_filename}\" \"${{CURRENT_PACKAGES_DIR}}/share/{lower_filename}\")\n"
```